price-feeder/oracle/filter_test.go

```diff
@@ -75,16 +89,16 @@ func TestSuccessFilterTickerDeviations(t *testing.T) {
 	}
 
 	providerTickers[config.ProviderBinance] = map[string]provider.TickerPrice{
-		pair.String(): atomTickerPrice,
+		pair.Base: atomTickerPrice,
```

Comment:
This PR also fixes these tests to be proper - these objects should be provider -> base -> price, but they were being used as provider -> [base+quote] -> price